auth/authpb/auth.proto

```diff
@@ -15,3 +15,10 @@ message User {
   bytes password = 2;
   int64 tombstone = 3;
 }
+
+// Role is a single entry in the bucket authRoles
+message Role {
+  int64 ID = 1;
```

Comment:
> Basically we allow user to have empty roles attached.

I didn't notice this behavior of v2.

However, the model would introduce unintentional permission configuration. At least we should provide an option of `role delete` that deletes a role from entire users, I think.